### DISCUSSION TITLE
backup and restore in command line

### DISCUSSION THREAD

**Discussion by Pachat** [Budibase/budibase#2837] [Help]
Is there a way to backup and restore apps using a command line?

**Comment by mjashanks** [comment]
No, we don't have that feature. You can Import/Export of apps, via the user interface - but I realise that this is not the same as Backup/Restore.

Out of interest, what is your reason for needing this? Is it
1. For disaster recovery?
2. To revert to previous versions?

You can access your database via `<your_domain_and_port>/db/_utils/# `. The username and password for this can be found in your `.env` file, wherever you are running budibase from. Then, have a look at https://docs.couchdb.org/en/latest/maintenance/backups.html on how to backup/restore couchDB databases.

Your individual apps are listed by their appId:
![image](https://user-images.githubusercontent.com/3524181/135598654-ad535fb2-3b2a-424e-a1f4-d1839b2c8a70.png)

`app_` are your published apps and `app_dev_` are the unpublished versions.
